frontend/styles/globals.css

```diff
@@ -2,6 +2,18 @@
 @tailwind components;
 @tailwind utilities;
 
+.button-box {
+  @apply w-full md:w-auto rounded-md px-3 py-2 text-sm font-semibold text-white shadow-sm focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2
+}
+
+.file-upload {
+  @apply relative cursor-pointer rounded-md bg-white font-semibold text-indigo-600 focus-within:outline-none focus-within:ring-2 focus-within:ring-indigo-600 focus-within:ring-offset-2 hover:text-indigo-500
```

Comment:
Don't use outline-none, it's necessary for keyboard navigation : https://accessibilite.numerique.gouv.fr/doc/RGAA-v4.1.pdf